src/findersync.cs

```diff
@@ -83,11 +83,12 @@ interface FIFinderSyncProtocol
 		[Export ("supportedServiceNamesForItemWithURL:")]
 		string[] SupportedServiceNames (NSUrl itemUrl);
 
+#if FALSE // TODO: Activate after 10.13 foundation APIs have been merged
```

Comment:
Whoops. I did file a bug, meant to add it to the comment. I'll update it with the bug #